ObjectState/TrackSends.cpp

```diff
@@ -270,30 +272,50 @@ void TrackSends::Build(MediaTrack* tr)
 			if (strncmp(line, searchStr, strlen(searchStr)) == 0) {
 	
 				// get the AUXVOL, AUXPAN, AUXMUTE subchunks
-				// note: chunks don't work properly with AIs currently
+				// note: envelope points in automations items aren't included in envelope chunks
 				// https://forum.cockos.com/showthread.php?t=205279
-				char AUXVOLchunk[4096]; char AUXPANchunk[4096]; char AUXMUTEchunk[4096];
-				WDL_FastString AUXVOLstr, AUXPANstr, AUXMUTEstr;
+				std::string AUXVOLstr, AUXPANstr, AUXMUTEstr;
 
 				TrackEnvelope* auxrcvVolEnv = (TrackEnvelope*)GetSetTrackSendInfo(pDest, -1, receiveIdx, "P_ENV", (void*)"<VOLENV");
 				TrackEnvelope* auxrcvPanEnv = (TrackEnvelope*)GetSetTrackSendInfo(pDest, -1, receiveIdx, "P_ENV", (void*)"<PANENV");
 				TrackEnvelope* auxrcvMuteEnv = (TrackEnvelope*)GetSetTrackSendInfo(pDest, -1, receiveIdx, "P_ENV", (void*)"<MUTEENV");
 
-				if (auxrcvVolEnv) {
-					GetEnvelopeStateChunk(auxrcvVolEnv, AUXVOLchunk, sizeof(AUXVOLchunk), true);
-					AUXVOLstr.Set(AUXVOLchunk);
+				if (auxrcvVolEnv) 
+				{
+					try {
+						AUXVOLstr = _GetEnvelopeStateChunkBig(auxrcvVolEnv);
+					} 
+					catch (_bad_get_env_chunk_big) {
+						
+						DisplayStoreSendEnvError(tr, "AUXVOL");
+						AUXVOLstr = "";

```

Comment:
`AUXVOLstr` is already initialized to an empty string by its constructor. There is no need re-assign it to a new empty value.